src/pages/ComponentsPage/ComponentsPage.js

```diff
@@ -1,8 +1,10 @@
 import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
 import Loader from '../../components/Loader/Loader';
 import './ComponentsPage.sass';
 import COMPONENTS from '../../app/components';
+import Sidebar from '../../components/Sidebar/Sidebar';
+import Searchbar from '../../components/Searchbar/Searchbar';
+import ComponentDocsList from '../../components/ComponentDocsList/ComponentDocsList';
```

Comment:
That can be in another PR, since we'll need to revisit the whole site. I agree, though